examples/siro_sandbox/config/hitl_defaults.yaml

```diff
@@ -0,0 +1,73 @@
+habitat_baselines:
+  # some defaults for interactive HITL applications
+  evaluate: True
+  num_environments: 1
+
+habitat_hitl:
```

Comment:
Every HITL app is required to inherit this config yaml and have a `habitat_hitl` block in its config. Note unlike habitat-lab and habitat-baselines, I'm not used Hydra structured configs. I don't know much about them yet. We could move to them later.